### DISCUSSION TITLE
Idle report do not include feedrate data "FS:0,0"

### DISCUSSION THREAD

**Discussion by Artemrull** [grblHAL/iMXRT1062#73] [Q&A]
Hi everybody. My Teensy 4.1 idle report do not include F,S data
<Idle|WPos:0.001,0.000,0.000,0.000,0.000|Bf:35,1023>
It is do not give to manage feedrate when CNC machine is run. What can I do for fix it?
I use GRBL Controller app for Android as G-code sender.

**Comment by terjeio** [comment]
Thats is due to beeing disabled by the $10 setting (Status report options) which has been extended to include related compile time options in legcay Grbl.
Default value is 511, which includes the FS element. Send `$$=10` to output the available flag values:
```
$10: Status report options as bitfield:
    0 - Position in machine coordinate (1)
    1 - Buffer state (2)
    2 - Line numbers (4)
    3 - Feed & speed (8)
    4 - Pin state (16)
    5 - Work coordinate offset (32)
    6 - Overrides (64)
    7 - Probe coordinates (128)
    8 - Buffer sync on WCO change (256)
    9 - Parser state (512)
    10 - Alarm substatus (1024)
    11 - Run substatus (2048)

Specifies optional data included in status reports.
If Run substatus is enabled it may be used for simple probe protection.

NOTE: Parser state will be sent separately after the status report and only on changes.
```

Or if you are running Windows you may try [ioSender or the Config.App](https://github.com/terjeio/ioSender/releases) for an easy to use settings UI:

![image](https://github.com/grblHAL/iMXRT1062/assets/20260062/d55ccf0d-0719-4b71-b041-b8ed1d15c52d)

**Reply by Artemrull** [reply]
I'm very grateful for your advice, it is worked!
Thanks!

**Comment by Artemrull** [comment]
Dear terjeio, all work very well until I kick a reset key. After reset value is  $10 =2, I need enter $10=511 again.
Maybe you know how fix this problem?

**Reply by terjeio** [reply]
This is really odd. Which reset key are you kicking:
* On the Teensy?
* Connected to the controller reset/estop/halt pin?
* In the sender?

If in the sender which one are you using? It might be the sender that changes the value.
Are any other setting values changed?
Does powering the controller off and on again change the setting?
How did you get the firmware? If you compiled yourself did you change _grbl/config.h_ in any way?

**Comment by Artemrull** [comment]
1. Which reset key are you kicking - In the sender
2. If in the sender which one are you using? - Grbl Controller, https://zeevy.github.io/grblcontroller/
3. Are any other setting values changed? -  no
4. Does powering the controller off and on again change the setting? - No, only reset is change settings
5. How did you get the firmware? -  Web builder

**Reply by terjeio** [reply]
I suspect it is the sender that modifies the setting. Try another, e.g. [ioSender](https://github.com/terjeio/ioSender/releases) if you have a Windows machine at hand.
